core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Aliasable.java

```diff
@@ -0,0 +1,5 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+public interface Aliasable {
+  public String getAlias();
```

Comment:
All the implementors of the interface have a setter, but all the users of the interface are read-only. So, I don't see the need.